src/AWSLambda/Events/APIGateway.hs

```diff
@@ -66,16 +97,17 @@ data ProxyRequestContext = ProxyRequestContext
   , _prcResourcePath :: !Text
   , _prcHttpMethod   :: !Text
   , _prcApiId        :: !Text
+  , _prcProtocol     :: !Text
```

Comment:
https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-mapping-template-reference.html

Reply:
Should be `Maybe Text` I think.